src/imagelightbox.css

```diff
@@ -126,6 +126,8 @@
     padding: 0.313em; /* 5 */
     transform: translateX(-50%);
     border-radius: 20px;
+    width:80%;
+    max-width: -moz-max-content;
 }
```

Comment:
yes, but autoprefixer adds prefixes to properties, but doesnt remove prefixes (like this "-moz" here). So your example would work on mozilla but not chrome :-(